src/resources/assets/js/mixins/largoContainer.vue

```diff
@@ -282,9 +281,29 @@ export default {
             if (!this.selectedLabel){
                 return []
             }
-            this.union = union ? 1 : 0;
-            this.getAnnotations(this.selectedLabel, filters, this.union);
+            this.loadFilters(this.selectedLabel.id, filters, union)
+        },
+
+        loadFilters(label, filters, union) {
+            if (filters.length == 0){
+                return
+            }
+            let cacheKey = JSON.stringify({...filters, label: label, union: union});
+            if (!this.filtersCache.hasOwnProperty(cacheKey)) {
+                Vue.set(this.annotationsCache, cacheKey, []);
```

Comment:
Should `annotationsCache` be modified here at all? If yes, is the `cacheKey` the correct one here? Before only the label ID was used as the cache key.